StandardCyborgUI/StandardCyborgUI/Bundle+SCUI.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension Bundle {
+    // Any entity using this prop should exist in the same target as the Bundle(for:) argument otherwise its contents
+    // won't be packaged correctly.
+    static let scuiBundle = Bundle(url: Bundle(for: ShutterButton.self).url(forResource: "StandardCyborgUI", withExtension: "bundle")!)!
```

Comment:
Hmm, did `Bundle(for: ShutterButton.self)` not return the right bundle?

Reply:
It didn't 😞 I tried that first and kept crashing. If you have another thought I'd be happy to try it since this isn't very pretty.